source/API/CMakeLists.txt

```diff
@@ -175,6 +175,8 @@ set(clang_headers_target ${CLANG_RESOURCE_PATH}/include)
 if(NOT LLDB_BUILT_STANDALONE)
   set(clang_headers_target clang-headers symlink_clang_headers)
 endif()
+set(SWIFT_RESOURCE_PATH "${LLDB_PATH_TO_SWIFT_BUILD}/lib${LLVMLIBDIR_SUFFIX}/swift")
+set(swift_headers_target "${SWIFT_RESOURCE_PATH}")
```

Comment:
`SwiftASTContext::GetResourceDir` was assuming that the `swift_dir` passed in would never be empty, but on Linux it would be empty because we now verify the `swift_dir` we compute. So I had to copy the swift resource directory into the lldb build tree.